src/App/components/Navbar/Navbar.styles.js

```diff
@@ -1,71 +1,254 @@
 import styled from "styled-components";
-import { Row, Icon } from "antd";
+import { Row, Input, Avatar } from "antd";
 
 export const NavContainer = styled(Row)`
-  background-color: #444;
+  background-color: #3e4257;
   width: 100%;
-  padding: 5px 0px 5px 200px;
+  height: 4.375rem;
+  display: flex;
   align-items: center;
+  font-family: Roboto;
 `;
+
 export const LogoContainer = styled.div`
-  display: flex;
-  align-items: center;
+  padding-left: 5px;
 `;
-export const LogoImg = styled.img`
-  width: 45px;
-  height: 45px;
-  margin-right: 10px;
+export const CurrentPage = styled.h3`
```

Comment:
Since we are already using `antd`, can we make use of the typography elements from `antd` to keep it consistent at all the places?
https://ant.design/components/typography/#header

Reply:
Sure, will implement them.

Reply:
Currently the `typography elements` are overriding my css styles so I've removed all the `heading h1,h3,h5 confustion` and put `paragraphs` everywhere. If we find a solution for the problem we will convert to `typography elements`.